e2e/cypress/integration/messaging/center_channel_rhs_overlap_spec.js

```diff
@@ -50,4 +88,806 @@ describe('Messaging', () => {
         // * Check if "Add Comment" button is visible
         cy.get('#addCommentButton').scrollIntoView().should('be.visible').and('have.value', 'Add Comment');
     });
+
+    it('MM-T712-1 Editing a post and hitting <enter> in code block saves post automatically instead of adding newline', () => {
```

Comment:
Should be fine but please use `_` before the step number for consistency.
```suggestion
    it('MM-T712_1 Editing a post and hitting <enter> in code block saves post automatically instead of adding newline', () => {
```